### PR TITLE
Request Spice

### PR DESCRIPTION
**Pull request by mistyLuminescence**
Have you been sitting in your department's foyer for 45 minutes with nothing to do? Do you want a little flavor added to the round? Are you too scared to adminhelp and go "give us something to do" because bwoink.ogg terrifies you?

If you answered 'yes' to one or more of the above questions, you may like the new verb - *Request Spice*. Available in the Admin tab (same place as Adminhelp), Request Spice will inform the admins that you want something interesting to happen- whether it's the sudden and inexplicable appearance of a giant spider in your office, a breach in a deserted room, or a visitor with critical injuries, you'll probably get something to do.

The spice button has a two minute cooldown when used, so no spammy-spammy. If you spammy-spammy anyway, expect the spice to be far too much for you.

After requesting spice, if your face and body itch as though insects were crawling over them, if your hands and feet swell dropsically, if you cannot stand the smell of food and bring it up after you have eaten it, if you feel as though you were going to be sick most of the time, if you experience weakness in the four limbs, if you have to go often to the latrine, or if your head or stomach violently ache—do not be alarmed or disturbed. All these effects are merely proofs that the spice you have requested is being delivered.

TL;DR: Bored? Now there's a button to gently nudge the admins to spice things up a little.

### CODE REVIEW

**Comment by Wickedtemp** [comment]
Might want to bump the cooldown up to like... ten minutes.

But yeh. I actually really like this. How many GM's do we have? Like five? This could be really neat for them.

**Comment by SpadesNeil** [comment]
You will all regret this.

I won't. :)
